everyvoice/tests/regression/test-demo-app-lj-full.py

```diff
@@ -0,0 +1,53 @@
+"""
+Test the Demo App with the regress-lj-full demo model as some exclusion options.
```

Comment:
I'm not sure I understand this sentence "as some exclusion options" ?

Reply:
Neither do I! I'll have to try to remember what I meant here... Must be a typo.